### DISCUSSION TITLE
Piece of cake

### DISCUSSION THREAD

**Discussion by NobleOsinachi** [NobleOsinachi/NobleOsinachi#120] [Q&A] [answered]
What are the two available methods for cloning a remote repository and what is the command to do so?

**Comment by vidardoes** [comment, marked as answer]
git clone repo_link

e.g. git clone https://github.com/NobleOsinachi/NobleOsinachi.git

**Comment by vidardoes** [comment]
You can use http or SSH

git clone https://github.com/NobleOsinachi/NobleOsinachi
